docs/remote_inference_blueprints/cohere_connector_image_embedding_blueprint.md

```diff
@@ -0,0 +1,324 @@
+  }
+}
+```
+
+It should return a response similar to this:
+
+```json
+{
+  "inference_results": [
+    {
+      "output": [
+        {
+          "name": "sentence_embedding",
+          "data_type": "FLOAT32",
+          "shape": [
+            1024
+          ],
+          "data": [
+            -0.0024547577,
+            0.0062217712,
+            -0.01675415,
+            -0.020736694,
+            -0.020263672,
+            ... ...
+            0.038635254
+          ]
+        }
+      ],
+      "status_code": 200
+    }
+  ]
+}
+```
+
+#### (Optional) 2. Setup k-NN index and ingestion pipeline
+
+##### 2a. Create your pipeline
+
+It is important that the `field_map` parameter contains all the document fields you'd like to embed as a vector. The key value is the document field name, and the value will be the field containing the embedding.
+
+```json
+PUT /_ingest/pipeline/cohere-ingest-pipeline
+{
+  "description": "Test Cohere Embedding pipeline",
+  "processors": [
+    {
+      "text_embedding": {
+        "model_id": "<MODEL_ID>",
+        "field_map": {
+          "passage_text": "passage_embedding"
+        }
+      }
+    }
+  ]
+}
+```
+
+Sample response:
+
+```json
+{
+  "acknowledged": true
+}
+```
+
+##### 2b. Create a k-NN index
+
+Here `cohere-nlp-index` is the name of your index, you can change it as needed.
+
+````json
+PUT /cohere-nlp-index
+
+{
+  "settings": {
+    "index.knn": true,
+    "default_pipeline": "cohere-ingest-pipeline"
+  },
+  "mappings": {
+    "properties": {
+      "id": {
+      "type": "text"
+      },
+      "passage_embedding": {
```

Comment:
How about changing to `image_embedding` ? 

And `passage_text` -> `image_base64`